pymongo/client_session.py

```diff
@@ -1062,11 +1075,14 @@ def mark_dirty(self) -> None:
         """
         self.dirty = True
 
-    def timed_out(self, session_timeout_minutes: float) -> bool:
-        idle_seconds = time.monotonic() - self.last_use
+    def timed_out(self, session_timeout_minutes: Optional[int]) -> bool:
+        if session_timeout_minutes is None:
+            return False
+        else:
+            idle_seconds = time.monotonic() - self.last_use
```

Comment:
nit: since the if-statement returns we don't need the `else`. The benefit is that:
1) code is easier to read with less indentation
2) there will be less code churn in the PR and in the git history.

What do you think?